exercises/practice/phone-number/.meta/lib/example.dart

```diff
@@ -13,27 +13,40 @@ class PhoneNumber {
       phoneNumber = phoneNumber.replaceAll(match.group(0)!, "");
     });
 
-    if (phoneNumber.contains(RegExp(r'[a-zA-Z]'))) throw FormatException('letters not permitted');
+    if (phoneNumber.contains(RegExp(r'[a-zA-Z]')))
+      throw FormatException('letters not permitted');
 
-    if (phoneNumber.contains(RegExp(r'!|@'))) throw FormatException('punctuations not permitted');
+    if (phoneNumber.contains(RegExp(r'!|@')))
+      throw FormatException('punctuations not permitted');
 
-    if (onlyDigits.length < 10) throw FormatException('incorrect number of digits');
+    if (onlyDigits.length < 10)
+      throw FormatException('must not be fewer than 10 digits');
 
-    if (onlyDigits.length > 11) throw FormatException('more than 11 digits');
+    if (onlyDigits.length > 11)
+      throw FormatException('must not be greater than 11 digits');
```

Comment:
I don't think a new line is needed here. I'll check the canonical data for the other change.